modules/vulnerabilities/unix/misc/jboss/files/jboss.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Vulnerable JBOSS server.
+
+[Service]
+ExecStart=/opt/jboss-6.1.0.Final/bin/run.sh
```

Comment:
If it's using a port > 1024, it should be able to run as a normal user rather than root